ktx2Decoder/src/transcoder.ts

```diff
@@ -0,0 +1,50 @@
+import { Nullable } from './types';
+import { WASMMemoryManager } from './wasmMemoryManager';
+import { KTX2FileReader, IKTX2_ImageDesc } from './ktx2FileReader';
+
+/**
+ * @hidden
+ */
+export enum sourceTextureFormat {
+    ETC1S,
+    UASTC4x4
+}
+
+/**
+ * @hidden
+ */
+export enum transcodeTarget {
+    ASTC_4x4_RGBA,
+    BC7_M5_RGBA,

```

Comment:
Consider renaming this to `BC7_RGBA`, as the old name is deprecated since https://github.com/KhronosGroup/KTX-Software/commit/571973e5330d7be209a537b108b450f4be34a689.

Reply:
Done